frontmacs-keys.el

```diff
@@ -87,8 +87,15 @@
 ;; Rename the current buffer and its visiting file if any.
 (global-set-key (kbd "C-c r") 'crux-rename-buffer-and-file)
 
-;; Kill whole line
-(global-set-key (kbd "C-w") 'crux-kill-whole-line)
+;; Kill whole line OR region
+(defun kill-region-or-line ()
```

Comment:
I'm thinking this should either be a `lambda` or defined in another file with a `frontmacs` prefix. Having a defun without a prefix makes it look like a builtin function. I like the idea of it being an interactive function though. 

maybe `frontmacs-kill-region-or-line` ?